client/src/components/FooterLinks/FooterLinks.tsx

```diff
@@ -53,6 +66,96 @@ export function FooterLinks() {
     );
   });
 
+  const handleOnChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    setNewsletterValue((prev) => ({
+      ...prev,
+      [name]: value,
+    }));
+  };
+  const handleError = (err: string) => {
+    console.log(err);
+    toast.error(err, {
+      position: "bottom-left",
+    });
+  };
+
+  const handleSuccess = (msg: string) => {
+    toast.success(msg, {
+      position: "bottom-left",
+    });
+  };
+
+  const handleNewsletterSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    try {
+      const { data } = await axios.post(
+        "http://localhost:3000/newsletter/signup",
+        {
+          ...newsletterValue,
+        },
+        { withCredentials: true }
+      );
+      const { success, message } = data;
+      if (success) {
+        handleSuccess(message);
+      } else {
+        handleError(message);
+      }
+    } catch (error) {
+      console.log(error);
+    }
+    setNewsletterValue({
+      ...newsletterValue,
+      email: "",
+      firstname: "",
+      lastname: "",
+    });
+  };
+
+  // Add the form as another group within the groups container
+  groups.push(
+    <div className={classes.wrapper} key="newsletter">
+      <form onSubmit={handleNewsletterSubmit} className={classes.form}>
+        <Text className={classes.title}>Subscribe to our newsletter</Text>
+        <div className={classes.formField}>
+          <label htmlFor="email" className={classes.label}>Email</label>
```

Comment:
I'd probably remove the labels and just keep the fields as "Email", "First Name", "Last Name" to minimize the footer.

Reply:
Some examples
![image](https://github.com/steventran619/tbmhp/assets/65416951/98a43949-eff6-4ce4-ac48-99d078194dd3)
![image](https://github.com/steventran619/tbmhp/assets/65416951/015588e0-3db0-4f58-a74e-bfd4a6c1bab2)
![image](https://github.com/steventran619/tbmhp/assets/65416951/d9cd7ca3-6443-46e7-9f77-3f1da000bc2b)
![image](https://github.com/steventran619/tbmhp/assets/65416951/9653329d-5d76-447e-aa12-179b5ec8b621)

Reply:
Good idea!